internals/overlord/state/change_test.go

```diff
@@ -589,21 +589,23 @@ var abortLanesTests = []struct {
 	abort  []int
 	result string
 }{
-
 	// Some basics.
 	{
 		setup:  "*:do",
 		abort:  []int{},
 		result: "*:do",
-	}, {
+	},
+	{
```

Comment:
This change seems to be the opposite of the test change above?

Reply:
I think that this is one of the spots that Ben mentioned in the PR description:

> The one change I wasn't happy about is the change from }, { between struct literals, to },\n{. But it seems it allows the one-line format sometimes, so I'm not sure if we can tweak that behaviour, or if it matters enough to worry about.
UPDATE: I discovered gofumpt is fine with this, we just had some inconsistent comments between struct entries. See [these commits](https://github.com/canonical/pebble/pull/723/files/f36d68da981f10aca367505a581d6fbfc1975118..8d6088c613d94a4beed06e2992a074954f7010bd) for the fix (we'll need to fix manually in a few other places too).

See the `// Some basics.` comment above. That comment doesn't appear to add significant value and could probably be removed.